Commentservice/src/main/java/com/nagarro/Commentservice/Exceptions/GlobalExceptionHandler.java

```diff
@@ -10,8 +10,12 @@
 @RestControllerAdvice
 public class GlobalExceptionHandler {
 
-	@ExceptionHandler({InvalidCommentException.class})
+	@ExceptionHandler({InvalidCommentException.class, CommentNotFoundException.class})
     public ResponseEntity<ErrorDTO> handlePostValidationException(Exception ex) {
```

Comment:
Method should be named appropriately , should be handleCommentValidationException()